lib/moon/design/dropdown.ex

```diff
@@ -19,6 +19,8 @@ defmodule Moon.Design.Dropdown do
   prop(testid, :string)
   @doc "Dropdown autoclose on click away"
   prop(autoclose, :boolean, default: true)
+  @doc "Well, disabled"
```

Comment:
```suggestion
  @doc "Disabled state for the dropdown"
```

Just in case you don't know yet, but you can suggest code changes in Github review. The PR author can apply your changes by pressing a single button

**Screenshot of suggestion button:**
<img width="744" alt="image" src="https://github.com/coingaming/moon/assets/19494863/f4afc365-7265-451d-9472-e80d272feee4">

Reply:
yes, and please always use it only for changes you're sure will work